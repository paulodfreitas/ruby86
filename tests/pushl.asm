.module m0
.pseg
        irmovl $100, %esp
	pushl %esp  
	popl  %eax
	halt

.end
