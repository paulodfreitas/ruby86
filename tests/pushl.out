@ 0000
30
84
00
01
00
00
a0
48
b0
08
10
@ 000b
